<ui version="4.0" >
 <class>TabnannyDialog</class>
 <widget class="QDialog" name="TabnannyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tabnanny Result</string>
  </property>
  <property name="whatsThis" >
   <string>&lt;b>Tabnanny Results&lt;/b>
&lt;p>This dialog shows the results of the tabnanny command. Double clicking an
entry will open an editor window and position the cursor at the respective line.&lt;/p></string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="resultList" >
     <property name="whatsThis" >
      <string>&lt;b>Result List&lt;/b>
&lt;p>This list shows the results of the tabnanny command. Double clicking
an entry will open this entry in an editor window and position the cursor at
the respective line.&lt;/p></string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Source</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="checkProgress" >
     <property name="toolTip" >
      <string>Shows the progress of the tabnanny action</string>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>resultList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
